src/github.com/getlantern/tarfs/tarfs.go

```diff
@@ -252,7 +252,7 @@ func (f *AssetFile) Readdir(count int) ([]os.FileInfo, error) {
 }
 
 func (f *AssetFile) Stat() (os.FileInfo, error) {
-	return f, nil
+	return &f.FakeFile, nil
```

Comment:
`AssetFile` embeds 3 members. Seems another member also has `Size()` method, which causes ambiguity, although I don't know exactly which one. Returning `&f.FakeFile` explicitly to eliminate the ambiguity, as `FakeFile` is existed to provided an `os.FileInfo` interface.